lib/editor_js/blocks/url_block.rb

```diff
@@ -0,0 +1,95 @@
+# frozen_string_literal: true
+
+module EditorJs
+  module Blocks
+    class UrlBlock < Base
+
+      # @TODO: supplement this whitelist from an ENV provided by the app,
+      # since it could be accessed from additional public URLs.
+      InternalHostWhitelist = [
+        'localhost', 'saleskit-staging.herokuapp.com'
```

Comment:
Do we want to be able to pass this in through an initializer for future use?

Reply:
Yeah, I would say let's punt that to a separate PR though so we can get this merged in.